SwiftUIDZ/SwiftUIDZ/ContentView.swift

```diff
@@ -7,13 +7,40 @@
 
 import SwiftUI
 
+/// Экран с таббаром
 struct ContentView: View {
+    
+    // MARK: - Body
+    
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        VStack {
+            TabView(selection: $currentTab) {
+                VPNView()
+                    .tag(TabType.vpn)
+                PurchaseView()
+                    .tag(TabType.purchase)
+                CartView()
+                    .tag(TabType.cart)
+            }
+            CustomTabBarView(currentTab: $currentTab)
+        }
+        .environmentObject(subscriptionOptions)
+    }
+    
+    // MARK: - Private Properties
+    
+    @StateObject private var subscriptionOptions = SubscriptionOptions()
```

Comment:
```suggestion
    @StateObject private var subscriptionOptions = SubscriptionOptions()
```
Разделяй враперы разного типа отступами